.github/workflows/community.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
           footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
 
-  statistics:
+
     runs-on: ubuntu-latest
```

Comment:
Please delete the whole block, `statistics` and all the nest info, up to line 33

Reply:
Deleted